### PR TITLE
Fibonacci

### PR DESCRIPTION
**Pull request by DamiXioR**
Fibonacci by Damixior.

### CODE REVIEW

File: module1/homework/fibonacci/fibonacci.hpp [hunk 1]
```diff
@@ -2,9 +2,33 @@
...
+
+    int temp = tab[1];
+    tab[1] = tab[0] + tab[1];
+    tab[0] = temp;
+
+    sequence--;
+
+    }
+
+    return tab[1];
```
Comment (kontrybutor) [inline module1/homework/fibonacci/fibonacci.hpp:29]: It seems you have an error in this line :)

File: module1/homework/fibonacci/fibonacci.hpp [hunk 2]
```diff
@@ -2,9 +2,33 @@
 
 int fibonacci_iterative(int sequence) {
 
+    int tab [] = {0,1};
+
+    if (sequence == 0) {
```
Comment (kontrybutor) [inline module1/homework/fibonacci/fibonacci.hpp:7]: You can simplify this few lines into:
```cpp
if (sequence <= 1)
{
    return sequence;
}

File: module1/homework/fibonacci/fibonacci.hpp (hunk 1 above)
Comment (DamiXioR) [reply module1/homework/fibonacci/fibonacci.hpp:29]: what king of error? Everything works

Comment (kontrybutor) [reply module1/homework/fibonacci/fibonacci.hpp:29]: Okay. My bad. You’re right.

File: module1/homework/fibonacci/fibonacci.hpp (hunk 2 above)
Comment (DamiXioR) [reply module1/homework/fibonacci/fibonacci.hpp:7]: :)

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 1430], "points": [2, 6] }

**Pull request closed without merging**